getting-started/web-app.md

```diff
@@ -0,0 +1,325 @@
+---
+title: Web App (Tutorial)
+page_title: First Steps with UI for Blazor in Web App 
+description: Make your first steps with Telerik UI for Blazor and build a Web App that runs the UI for Blazor components.
+slug: getting-started/web-app
+tags: get,started,first,steps,web,app,template
+published: true
+position: 4
+---
+
+# First Steps with UI for Blazor in Web App
+
+This article explains how to get the Telerik UI for Blazor components in your <a href = "https://learn.microsoft.com/en-us/aspnet/core/blazor/project-structure?view=aspnetcore-8.0#blazor-web-app" target="_blank">**Blazor Web App** Blazor</a> project and start using them quickly. You will create a new application from scratch, learn how to add the UI for Blazor components to a project, and finally, add a UI component to a view.
```

Comment:
.NET 8 should be present somewhere in the first paragraph, so that it's immediately clear what the article is about.

Consider also adding .NET 8 to the page_title and description.

```suggestion
This article explains how to use the Telerik UI for Blazor components in a <a href = "https://learn.microsoft.com/en-us/aspnet/core/blazor/project-structure?view=aspnetcore-8.0#blazor-web-app" target="_blank">.NET 8 Blazor Web App</a> project template. You will create a new application from scratch, learn how to add the UI for Blazor components to a project, and finally, add a UI component to a view.
```